src/entities/station.entity.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  Entity,
+  PrimaryGeneratedColumn,
+  Column,
+  CreateDateColumn,
+  UpdateDateColumn,
+  DeleteDateColumn,
+  ManyToOne,
+  JoinColumn,
+} from 'typeorm';
+import { State } from './state.entity';
+import { City } from './city.entity';
+
+/**
+ * Represents Station
+ */
+@Entity()
+export class Station {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column({ nullable: true })
+  description: string;
+
+  @Column()
+  address: string;
+
+  @Column()
+  area: string;
+
+  @Column()
+  pincode: number;
+
+  @Column({ default: true })
+  open: boolean;
+
+  @Column()
+  managerName: string;
+
+  @Column()
+  mobileNumber1: string;
+
+  @Column({ nullable: true })
+  mobileNUmber2: string;
+
+  @Column({ nullable: true })
+  landlineNumber: string;
+
+  @Column()
+  email: string;
+
+  @Column({ nullable: true })
+  carpetArea: string;
+
+  @ManyToOne(() => State)
```

Comment:
Had commented last time too.

Add a field for cityId and stateId.
ManyToOne is just for foreign key